src/components/flat-table/__internal__/use-calculate-sticky-cells.ts

```diff
@@ -28,5 +31,8 @@ export default (id: string) => {
     onKeyDown,
     isFirstCell,
     isExpandableCell,
+    highlighted,
```

Comment:
**comment**: I understand why you've added these props here, but it seems to me that this hook is now about much more than just "sticky cells". Maybe rename it to something more generic like `useTableCell`? (But there's likely something much better than that.)

Reply:
valid point I'll have a little think about it